app/code/Magento/Checkout/view/frontend/web/js/model/quote.js

```diff
@@ -77,7 +77,7 @@ define([
          * @return {*}
          */
         getItems: function () {
-            return window.checkoutConfig.quoteItemData;
+            return totals.getItems();
```

Comment:
This seems to be a correct solution, however did you make sure that data returned from this method is exactly the same in all cases?